CONTRIBUTING.rst

```diff
@@ -62,8 +62,26 @@ but you can also use a ``virtualenv`` of your own creation.
 
 .. _repo: https://github.com/GoogleCloudPlatform/google-cloud-python
 
-Using a custom ``virtualenv``
-=============================
+Use ``nox``, not ``virtualenv``
```

Comment:
Can someone explain why we need this section at all? There's instructions below on how to use Nox.

Reply:
+1
Using virtualenv directly creates problems, so I think we should just say to use nox, and drop this section.